### ISSUE TITLE
Flu vaccine distances

### ISSUE THREAD

**Issue by trvrb** [nextstrain/augur#98]
@sidneymbell ---

Thank you for working on the flu vaccine distances. These numbers would be generally good to have moving forward and I'd really like things incorporated into augur. Do you think you could do this? I thought it would work to make a new function in `flu.process.py` to export a JSON file to `auspice/` that contains all the relevant data. We want every tip in the tree to have measurements against all available vaccine strains and use three different metrics. This is what I was picturing:

```
[
  {
    "strain": "A/Incheon/677/2006",
    "num_date": 2006.867,
    "region": "japan_korea",
    "ep": {
      "A/Perth/16/2009": 4,
      "A/Victoria/361/2011": 6
    },
    "cTiter": {
      "A/Perth/16/2009": 2.3,
      "A/Victoria/361/2011": 4.2
    },
    "cTiterSub": {
      "A/Perth/16/2009": 1.9,
      "A/Victoria/361/2011": 3.7
    }
  }
]
```

This would then make a simple lookup table for downstream analyses. I could work from this output as could Frank.

I didn't want to include this in the `tree.json` as it felt like it would add unnecessary cruft.

cc @huddlej

**Comment by sidneymbell** [comment]
Sure, I'd be happy to. But, as you noted in your email,  
>>>Antigenic distance...should be the average of tips in a clade that are circulating in a given season  
Thus, these values of "antigenic distance" are actually time-dependent.   
E.g., let's say clade A has 10 descendants in the tree. 3 of these tips are from 2014-2015, while 7 are from 2015-2016. The mean antigenic distance for 2014-2015 clade A descendants to vaccine will be different than the mean antigenic distance for 2015-2016 clade A descendants vs. vaccine.

**Comment by trvrb** [comment]
Yes. Antigenic distances between vaccine strains and clades will be time-dependent. Here, I was thinking to just output antigenic distances between vaccine strains and individual tips. These pairwise measurements do not suffer from time dependence. Binning into seasons, etc... can happen downstream.

Although... the other approach would be to do that seasonal binning here. Fairly agnostic on this point.

**Comment by sidneymbell** [comment]
Ahh, yes. Can do. I'll finish writing hour and then start on that later this afternoon.  
  
Ehh, the seasonal binning is delicate enough I'd rather do it downstream where there are fewer moving parts and I can clearly see what's happening / breaking.

**Comment by trvrb** [comment]
This issue was resolved by PR #101.
